### DISCUSSION TITLE
Inquiry: Availability of Blobfuse2 on Microsoft's Package Manager

### DISCUSSION THREAD

**Discussion by mbagiella** [Azure/azure-storage-fuse#1417] [General]
Dear Community,

I'm eager to learn more about the official publication plans for Blobfuse2 on Microsoft's package manager. As an enthusiastic user of Blobfuse2, I believe its availability on a package manager directly from Microsoft would greatly enhance accessibility and ease of installation for users across various platforms.

Could someone shed some light on whether Microsoft has plans to officially publish Blobfuse2 on their package manager? If so, is there an estimated timeline for when this might happen? Alternatively, if there are any discussions or announcements regarding this matter, I'd appreciate any insights shared.

Thank you in advance for your assistance and insights!

Best regards,
Mattia

**Comment by bhansalivikas** [comment]
Hi Mattia,

We already publish Blobfuse2 on Microsoft linux repositories for various linux distros like Ubuntu, RHEL, Mariner etc. Is there anything else you are looking for here?

Regards
Vikas Bhansali
________________________________
From: Mattia Bagiella ***@***.***>
Sent: 16 May 2024 19:55
To: Azure/azure-storage-fuse ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [Azure/azure-storage-fuse] Inquiry: Availability of Blobfuse2 on Microsoft's Package Manager (Discussion #1417)


Dear Community,

I'm eager to learn more about the official publication plans for Blobfuse2 on Microsoft's package manager. As an enthusiastic user of Blobfuse2, I believe its availability on a package manager directly from Microsoft would greatly enhance accessibility and ease of installation for users across various platforms.

Could someone shed some light on whether Microsoft has plans to officially publish Blobfuse2 on their package manager? If so, is there an estimated timeline for when this might happen? Alternatively, if there are any discussions or announcements regarding this matter, I'd appreciate any insights shared.

Thank you in advance for your assistance and insights!

Best regards,
Mattia

**Reply by mbagiella** [reply]
Good to know.
With RHEL9.3, what's the command to get blobfuse2 ?
When I do sudo yum install fuse3 fuse3-libs blobfuse2, it installs fuse3 and fuse3-libs but blobfuse2 seems missing, it is unable to find a match

**Reply by bhansalivikas** [reply]
Kindly follow this page: https://github.com/Azure/azure-storage-fuse/wiki/Blobfuse2-Installation

**Reply by mbagiella** [reply]
Thanks, I found it and was also able to install it.
I did an update of the package manager by executing this command : sudo rpm -Uvh https://packages.microsoft.com/config/rhel/9/packages-microsoft-prod.rpm
